fdbserver/fdbserver.actor.cpp

```diff
@@ -325,7 +325,7 @@ UID getSharedMemoryMachineId() {
 	loop {
 		try {
 			// "0" is the default parameter "addr"
-			boost::interprocess::managed_shared_memory segment(boost::interprocess::open_or_create, "fdbserver", 1000, 0, p.permission);
+			boost::interprocess::managed_shared_memory segment(boost::interprocess::open_or_create, "fdbserver_boost_1_67", 1000, 0, p.permission);
```

Comment:
Can we make this format string `format("fdbserver_%s", BOOST_LIB_VERSION)` where `BOOST_LIB_VERSION` is defined by boost[1]?

[1] https://www.boost.org/doc/libs/1_45_0/boost/version.hpp

Reply:
Sure.